yt_dlp/extractor/wikimedia.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
```

Comment:
remove

Reply:
thanks for help fixed.  moving to next issue.